view.rb

```diff
@@ -1,74 +1,94 @@
 module Dialog
-	### Asks if you have an account already.
-	def self.new_or_old_user
-		print "\nHello! Do you have a pre-existing account with us? ('yes' or 'no')\n"
+	def self.new_or_old_user # Asks if you have an account already.
+		print "\nHello! Do you have a pre-existing account with us? ('yes' or 'no'): "
 		gets.chomp.downcase
 	end
 
-	### Initial greeting (first thing user should see straight after entering ruby <file> into the command line)
-	def self.greeting  
-		puts "\nWelcome to RubyBank, where all of your financial dreams come true."
-		puts "\nWould you like to make an account? ('yes' or 'no')" 
+	def self.greeting # Initial greeting (first thing user should see straight after entering ruby <file> into the command line)
+		print "\nWelcome to RubyBank, where all of your financial dreams come true."
+		print "\n\nWould you like to make an account? ('yes' or 'no'): " 
 		gets.chomp.downcase
 	end
 
-	### Says goodbye and mourns the loss of user.
-	def self.goodbye 
-		puts "\nThat is too bad. We are a great bank. Suit yourself! Have a nice day!\n"
-	end
-	### Explains that method is not yet available, check back later.
-	def self.goodbye2			
-		puts "\nThis is not yet a current working method in RubyBank. Check back in a few hours.\nGoodbye!\n\n"
+	def self.goodbye # Says goodbye and mourns the loss of user.
+		print "\nThat is too bad. We are a great bank. Suit yourself! Have a nice day!\n\n"
 	end
 
 	def self.goodbye_cust # The goodbye message for a customer. 
-		puts "\nHave a great day! We will look forward to your return!\n"
+		print "\nHave a great day! We will look forward to your return!\n\n"
 	end
 
 	def self.account_name # this should be the dialog around initializing your bank account. You give your account name.	
-		puts "\nplease enter your the name you wish to associate with the account."
+		print "\nPlease enter your the name you wish to associate with the account: "
 		gets.chomp.downcase
```

Comment:
I think ultimately you don't want to use downcase on this because you want to be able to capitalize names that get put in the database. But if you want to leave it this way for now to make typing during testing faster that's fine.